README.md

```diff
@@ -1,65 +1,93 @@
-# Welcome to Streamlit :wave:
+![Streamlit logo|200x200](https://i.ibb.co/5FdSr1V/streamlit.png)
```

Comment:
Love the logo. There could be a tiny bit more padding above it 😉 Maybe add an empty line? Or edit the image slightly?

Reply:
Added a line